### PR TITLE
PagerDuty - 2 to 3

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PagerDuty/Integrations/PagerDuty/PagerDuty.py [hunk 1]
```diff
@@ -116,7 +113,7 @@ def http_request(method, url, params_dict=None, data=None, json_data=None, addit
         raise
 
 
-def translate_severity(sev):
+def translate_severity(sev):  # pragma: no cover
```
Comment (tkatzir) [inline Packs/PagerDuty/Integrations/PagerDuty/PagerDuty.py:116]: Need a unit test

Comment (tkatzir) [reply Packs/PagerDuty/Integrations/PagerDuty/PagerDuty.py]: The fact that `'high'` is all lower case, and `'Low'` is camel case raises a brow.
I'd check against a `sev.lower()`

**Review by tkatzir — COMMENTED** [review]
Something doesn't look right with `translate_severity`.

**Comment by MLainer1** [comment]
Decided to leave the function `translate_severity` as is until more research can be done (currently we do not have valid credentials)

**Review by dantavori — APPROVED** [review]
as discussed, make sure the translate_severity is not a bug. if not - let's keep it, otherwise change it to what @tkatzir suggested. approving, but don't merge before checking.

**Comment by MLainer1** [comment]
Found an indication in the API documentation about possible values for the urgency entry:
image
https://developer.pagerduty.com/api-reference/005299ed43553-get-an-incident
Seems as a bug, changing it in this pr

**Review by tkatzir — APPROVED** [review]

**Comment by MLainer1** [comment]
This PR needs a force merge due to not working credentials

**Pull request merged**